webapp/client/cypress/integration/noPauschbetragPersonA.spec.js

```diff
@@ -0,0 +1,126 @@
+    // There is no possible combination to only have a fahrkostenpauschbetrag claim.
+    beforeEach(() => {
+      cy.request("POST", "/testing/set_data/form_data", {
+        person_a_has_disability: "yes",
+        person_a_has_pflegegrad: "no",
+        person_a_disability_degree: 60,
+      });
+      cy.visit("/lotse/step/person_a_no_pauschbetrag");
+    });
+
+    it("Should redirect to person a merkzeichen page", () => {
+      cy.url().should("include", "/lotse/step/familienstand");
+      // TODO cy.url().should("include", "/lotse/step/merkzeichen_person_a");
+    });
+  });
+
+  context(
+    "with person a has pauschbetrag and fahrkostenpauschbetrag claim",
+    () => {
+      beforeEach(() => {
+        cy.request("POST", "/testing/set_data/form_data", {
+          person_a_has_disability: "yes",
+          person_a_has_pflegegrad: "no",
+          person_a_disability_degree: 80,
+          person_a_has_merkzeichen_g: true,
+        });
+        cy.visit("/lotse/step/person_a_no_pauschbetrag");
+      });
+
+      it("Should redirect to person a merkzeichen page", () => {
+        cy.url().should("include", "/lotse/step/familienstand");
+        // TODO cy.url().should("include", "/lotse/step/merkzeichen_person_a");
+      });
+    }
+  );
+
+  context(
+    "with person a has no pauschbetrag or fahrkostenpauschbetrag claim",
+    () => {
+      beforeEach(() => {
+        cy.request("POST", "/testing/set_data/form_data", {
+          person_a_has_disability: "yes",
+          person_a_has_pflegegrad: "no",
+        });
+        cy.visit("/lotse/step/person_a_no_pauschbetrag");
+      });
+
+      it("Should not redirect", () => {
+        cy.url().should("include", "/lotse/step/person_a_no_pauschbetrag");
+      });
+
+      it("Should link back to merkzeichen page", () => {
+        cy.get("a").contains("Zurück").click();
+        cy.url().should("include", "/lotse/step/person_a_has_disability");
+        // TODO cy.url().should("include", "/lotse/step/merkzeichen_person_a");
+      });
+
+      it("Should link forward to telephone number page", () => {
+        cy.get("a").contains("Weiter").click();
+        cy.url().should("include", "/lotse/step/telephone_number");
+      });
+
+      context("With joint taxes", () => {
+        beforeEach(() => {
+          cy.request("POST", "/testing/set_data/form_data", {
+            familienstand: "married",
+            familienstand_date: "02.09.2000",
+            familienstand_married_lived_separated: "no",
+            familienstand_confirm_zusammenveranlagung: true,
+            person_a_has_disability: "yes",
+            person_a_has_pflegegrad: "no",
+          });
+          cy.visit("/lotse/step/person_a_no_pauschbetrag");
+        });
+
+        it("Should link forward to person b page", () => {
+          cy.get("a").contains("Weiter").click();
+          cy.url().should("include", "/lotse/step/person_b");
+        });
+      });
```

Comment:
Personally, I think this context should be outside the other context because it does not keep any of the data. However, if you think that this makes more sense because it structures the tests better, that would be fine with me